front/pages/api/w/[wId]/vaults/[vId]/contentHandler.ts

```diff
@@ -0,0 +1,83 @@
+import type { ContentNodeType, WithAPIErrorResponse } from "@dust-tt/types";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import {
+  getManagedDataSourceContent,
+  getUnmanagedDataSourceContent,
+} from "@app/lib/api/vaults";
+import type { DataSourceResource } from "@app/lib/resources/data_source_resource";
+import { apiError } from "@app/logger/withlogging";
+
+export type LightContentNode = {
```

Comment:
Let's avoid exporting types when they are not needed. If we plan to use it in the UI, then it should live in types.